src/app/components/character-count/index.html

```diff
@@ -0,0 +1,284 @@
+        >
+          Don't use when
+        </h3>
+
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Don’t use character count on inputs where the character limit is apparent or implied by the data type.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-2.png"
+              />
+              <div
+                class="caption caption-danger sky-text-danger"
+              >
+                Don't use character count when the content has an expected or standard length.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Don’t use character count on inputs when exceeding the character limit is unrealistic. Use normal form validation to handle errors caused by character limit violations.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div
+              class="sky-margin-stacked-content-section"
+            >
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-3.png"
+              />
+              <div
+                class="caption caption-danger sky-text-danger"
+              >
+                Don't use character count when the character limit is unlikely to be exceeded.
+              </div>
+            </div>
+
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-4.png"
+              />
+              <div
+                class="caption caption-success sky-text-success"
+              >
+                Do use normal form validation for edge cases where a high character limit is exceeded.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+
+      <sky-demo-page-content sectionHeading="Anatomy">
+
+        <sky-docs-anatomy
+          imagePath="~/assets/img/guidelines/character-count/character-count-anatomy-1.png"
+        >
+          <sky-docs-anatomy-item
+            anatomyNumber="1"
+          >
+            Input field
+          </sky-docs-anatomy-item>
+          <sky-docs-anatomy-item
+            anatomyNumber="2"
+          >
+            Character count/limit
+          </sky-docs-anatomy-item>
+          <sky-docs-anatomy-item
+            anatomyNumber="3"
+          >
+            Danger icon
+          </sky-docs-anatomy-item>
+          <sky-docs-anatomy-item
+            anatomyNumber="4"
+          >
+            <a routerLink="/components/status-indicator"> Status indicator</a> (danger)
```

Comment:
It seems weird to only refer to this as `Status indicator` since I think it applies to the red border, shadow, and error message as well, not just the indicator at the far bottom left. If it truly is just the status indicator, then we should move the #4 in the image to the fat bottom left too. But if not, I think we need a different term here. Maybe something more like `Exceeded limit warning` or `Validation for exceeded limit`. Not sure exactly what it should be, but I think the current descriptor is too specific.

Reply:
One objective of the Anatomy documentation is to highlight how components build and compose using other components. Being specific about using the actual status indicator class, and not just an error message that looks like one, is part of that objective. I'm inclined to leave it called out that way, and to leave the callout bubble to the right of the text as well (since that's an anatomical part of the status indicator itself).